copter/source/docs/circle-mode.rst

```diff
@@ -13,21 +13,39 @@ of the vehicle with the nose of the vehicle pointed at the center.
    Mission Planner reports the units as cm for all versions of the
    code.
 
-Setting the CIRCLE_RADIUS to zero will cause the copter to simply stay
+Setting the :ref:`CIRCLE_RADIUS<CIRCLE_RADIUS>` to zero will cause the copter to simply stay
 in place and slowly rotate (useful for panorama shots).
 
 The speed of the vehicle (in deg/second) can be modified by changing the
-CIRCLE_RATE parameter.  A positive value means rotate clockwise, a
+:ref:`CIRCLE_RATE<CIRCLE_RATE>` parameter.  A positive value means rotate clockwise, a
 negative means counter clockwise.  The vehicle may not achieve the
 desired rate if this requires the acceleration towards the center of the
-circle to surpass the maximum acceleration held in the WPNAV_ACCEL
+circle to surpass the maximum acceleration held in the :ref:`WPNAV_ACCEL<WPNAV_ACCEL>`
 parameter (units are cm/s/s).
 
+The circle rate set above can be dynamically adjusted in flight be two methods. The first is the use of RC Channel 6 if the :ref:`TUNE<TUNE>` option is set to 39, allowing decreasing the rate 50% or increasing it by 100%, at the channel min and max. The other is by enabling the :ref:`CIRCLE_CONTROL<CIRCLE_CONTROL>` parameter to allow stick adjustment of radius and speed.
+
+Circle Control Option
+=====================
+
+When enabled, the :ref:`CIRCLE_CONTROL<CIRCLE_CONTROL>` parameter allows the adjustment of the circle's radius and angular velocity.
+
+- Pitch stick down (negative) reduces the radius until it reaches zero.
+- Pitch stick up (positive) increases the radius
```

Comment:
The up/down was reversed in a subsequent PR. Pitch up, as in forward from an FPV perspective, reduces the radius. pitch down, as in reverse from an FPV perspective, increases the radius.